src/data/map_gfx.asm

```diff
@@ -7,239 +7,239 @@ MACRO map_gfx
 ENDM
 
 MapGfx:
-	; OVERWORLD_MAP
-	map_gfx TILEMAP_000, TILESET_000, PALETTE_000, OW_ANIM_00,   PALETTE_166
+	; MAP_00
```

Comment:
I stuck with "MAP_##" since the tileset labels already state the location of each map. Tho if anything that label should be "OVERWORLD_MAP_TCG", with "MAP_003" being "OVERWORLD_MAP_GR".

Reply:
(Fixed as of Feedback Fix No2)